Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM gradle:5.6.2-jdk8
```

Comment:
I run the dockerfile. I seems a bit large. I'm not an expert on Docker, but can't we reduce final size with:
`FROM gradle:5.6.2-jdk8 as BUILD`
Thank you very much for contributing.

Reply:
Using [Multi-stage build](https://docs.docker.com/develop/develop-images/multistage-build/) will not reduce current image size as far as I know but I'll ad it in case this ` Dockerfile` can be extended later to create a production image to run the site